Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Error.swift

```diff
@@ -150,9 +150,17 @@ extension PathHierarchy.Error {
             let solutions: [Solution] = candidates
                 .sorted(by: collisionIsBefore)
                 .map { (node: PathHierarchy.Node, disambiguation: String) -> Solution in
-                    return Solution(summary: "\(Self.replacementOperationDescription(from: disambiguations.dropFirst(), to: disambiguation)) for\n\(fullNameOfNode(node).singleQuoted)", replacements: [
-                        Replacement(range: replacementRange, replacement: "-" + disambiguation)
-                    ])
+                    // Suggest removing the disambiguation hash entirely
+                    if disambiguation.isEmpty {
+                        return Solution(summary: "\(Self.replacementOperationDescription(from: disambiguations, to: disambiguation)) for\n\(node.name.singleQuoted)", replacements: [
```

Comment:
There's an opportunity to customize the name in `PathHierarchyBasedLinkResolver/fullName(of:in:)`. 

In the general case we'd still want to display the full declaration. AFAIK, only overload groups want to display a different name. We could make that customization there instead.

Reply:
Yes good idea - added in 3492fcc72328dc5900c24373b79339e95733e12e.